rstream/client.py

```diff
@@ -172,9 +173,34 @@ async def start(self) -> None:
         self._conn = Connection(self.host, self.port, self._ssl_context)
         await self._conn.open()
         self.start_task("listener", self._listener())
+
         self.add_handler(schema.Heartbeat, self._on_heartbeat)
         self.add_handler(schema.Close, self._on_close)
 
+    async def run_queue_listener_task(self, subscriber_name: str, handler: HT[FT]):
```

Comment:
nit: if function doesn't return a value, it should be annotated as `-> None` 
PS I think maybe we should run mypy in `strict` mode, so we don't have any typing issues (of course that's not for this PR)

Reply:
OK!